dom0/sd-log-reboot.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think this is required anymore, given the `qvm-shutdown sd-log` call you make in `./scripts/provision-all`. Am I missing something?

Reply:
We could also run `qvm-shutdown && qvm start` if we need it to be up, instead of having to maintain this complex logic